corehq/apps/domain/templates/domain/admin/sms_settings.html

```diff
@@ -7,28 +7,21 @@
 {% block page_content %}
   {% initial_page_data "settings" settings %}
   {% initial_page_data "other_sms_codes" other_sms_codes %}
-  <form id="settings" class="form form-horizontal" method="post">
+  <form id="settings" method="post">
     {% csrf_token %}
     <fieldset>
       <legend>{% trans 'Stock Actions' %}</legend>
-      <div class="form-group">
-        <label class="{% css_label_class %} control-label">{% trans 'Edit Actions' %}</label>
-        <div class="{% css_field_class %}">
-          {% include "domain/admin/commtrack_action_table.html" %}
-          <button type="button" class="btn btn-default" data-bind="click: new_action">
-            <i class="fa fa-plus"></i> {% trans "New Action" %}
-          </button>
-        </div>
+      <div class="{% css_field_class %}">
+        <button type="button" class="btn btn-outline-primary float-end mb-3" data-bind="click: new_action">
+          <i class="fa fa-plus"></i> {% trans "New Action" %}
+        </button>
+        {% include "domain/admin/commtrack_action_table.html" %}
```

Comment:
The table moved from above the "New Action" button to below it. Is that a style guide recommendation, or was it possibly unintentional?

I assume new rows are added at the bottom of the table, so it seems possibly more intuitive to have the button below the table.

Reply:
It was intentional, because it's more common in HQ to have "Add" buttons at the top of the page. But, thinking about it, I'm thinking of pages like conditional alerts, where adding a new item takes you to a new page, rather than adding a row to a table on the same page. Moved it back in 6f545cde25b

![Screenshot 2024-03-20 at 4 39 11 PM](https://github.com/dimagi/commcare-hq/assets/1486591/58bdbc8e-2613-44e4-954e-80db9c1fc781)